apps/meteor/app/livechat/server/api/lib/livechat.js

```diff
@@ -39,15 +39,17 @@ export function findDepartments(businessUnit) {
 }
 
 export function findGuest(token) {
-	return LivechatVisitors.getVisitorByToken(token, {
-		fields: {
-			name: 1,
-			username: 1,
-			token: 1,
-			visitorEmails: 1,
-			department: 1,
-		},
-	});
+	return Promise.await(
```

Comment:
most places I found using `findGuest` were already `async`, so changing this to `async` would be an easy "enough" change